app/scripts/services/authorization.js

```diff
@@ -96,16 +96,22 @@ angular.module("openshiftConsole")
       return _.get(cachedRulesByProject.get(projectName || currentProject), ['rules']);
     };
 
+    var checkRulesForResourceVerb = function(rules, resource, verb) {
+      return _.contains(rules[resource], verb) || _.contains(rules[resource], '*');
+    };
+
     var canI = function(verb, resource, projectName) {
       projectName = projectName || currentProject;
       var rules = getRulesForProject(projectName);
       if (permissiveMode) {
         return true;
       } else if (rules) {
         if (rules[resource]) {
-          return _.contains(rules[resource], verb) || _.contains(rules[resource], '*');
+          return checkRulesForResourceVerb(rules, resource, verb);
         } else if (rules['*']) {
-          return _.contains(rules['*'], verb) || _.contains(rules['*'], '*');
+          return checkRulesForResourceVerb(rules, '*', verb);
+        } else if (rules['*/*']) {
+          return true;
```

Comment:
I don't think this is right. We should at least check if it contains `verb` or `'*'`.

@deads2k Can you help me understand the difference between `rules['*']` and `rules['*/*']`?